torch/distributions/kl.py

```diff
@@ -313,6 +315,13 @@ def _kl_transformed_transformed(p, q):
     return kl_divergence(p.base_dist, q.base_dist)
 
 
+@register_kl(LogisticNormal, LogisticNormal)
```

Comment:
This should not be needed. `kl_divertence(LogisticNormal(...), LogisticNormal(...))` should come for free because it inherits from `TransformedDistribution`. Could you remove this registration?